OutOfSchool/OutOfSchool.Encryption/Services/IEUSignOAuth2Service.cs

```diff
@@ -0,0 +1,24 @@
+using OutOfSchool.Encryption.Models;
+
+namespace OutOfSchool.Encryption.Services;
+
+public interface IEUSignOAuth2Service
```

Comment:
I suppose we can add the documentation for interface as well )
And: we might use the `/// <inheritdoc/>` in all the implementations for each method so they will be documented.